example/bin/console.php

```diff
@@ -22,8 +22,7 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
-/** @var EntityManager $em */
-$em = include __DIR__.'/../em.php';
+include __DIR__.'/../em.php';
```

Comment:
IMO, we should add the missing `return` statement for the `$em` variable here: 
https://github.com/doctrine-extensions/DoctrineExtensions/blob/25c94bc3e526972c0d0545cf0ded2447e9de00a2/example/em.php#L181

See [include and the return statement](https://www.php.net/manual/en/function.include.php#example-143).